server/data.py

```diff
@@ -243,15 +244,17 @@ def write_variants(self, variants, filters):
         chips = self._get_chips(chr, pos, ref, alt) if len(vars_data) == 1 else set()
         data = self._count_gt_for_write(variants.split(','), vars_data, filters, chips)
         filename = variants.replace(',', '_') + '__' + '_'.join([k+'_'+v for k,v in filters.items()]) + '.txt'
-        path = os.sep.join([self.conf['write_dir'], filename])
         try:
-            if not os.path.exists(self.conf['write_dir']):
-                os.makedirs(self.conf['write_dir'])
-            data.to_csv(path, sep='\t', index=False, na_rep='NA')
+            data.to_csv(sep='\t', index=False, na_rep='NA')
+            output = make_response(data.to_csv(sep='\t', index=False, na_rep='NA'))
+
+            output.headers["Content-Disposition"] = "attachment; filename=" + filename
+            output.headers["Content-type"] = "text/csv"
```

Comment:
Should this be text/tab-separated-values or is it better to have csv?

Reply:
Good point. Updated both content-type & filename